content/id/docs/concepts/overview/object-management-kubectl/imperative-command.md

```diff
@@ -0,0 +1,132 @@
+
+*Tool* `kubectl` juga mendukung perintah-perintah pembuatan objek yang berdasarkan pada tipe objek. Perintah-perintah ini mendukung lebih banyak tipe objek dan lebih eksplisit tentang intensi mereka. Tapi, perintah-perintah ini memerlukan pengguna untuk memahami tipe dari objek-objek yang hendak mereka buat.
+
+- `create <objecttype> [<subtype>] <instancename>`
+
+Beberapa tipe objek memiliki sub tipe yang bisa kamu spesifikasikan pada perintah `create`. Misalnya, objek Service memiliki beberapa sub tipe seperti ClusterIP, LoadBalancer, dan NodePort. Berikut adalah sebuah contoh cara membuat sebuah Service dengan sub tipe NodePort:
+
+```shell
+kubectl create service nodeport <myservicename>
+```
+
+Pada contoh di atas, perintah `create service nodeport` merupakan sub perintah dari `create service`.
+
+Kamu bisa menggunakan *flag* `-h` untuk mencari argumen-argumen dan paramenter-parameter yang didukung oleh sebuah sub perintah:
+
+```shell
+kubectl create service nodeport -h
+```
+
+## Cara memperbarui objek
+
+Perintah `kubectl` mendukung perintah-perintah berbasis kata kerja untuk beberapa operasi pembaruan yang umum. Perintah-perintah ini diberi nama yang memudahkan pengguna yang belum familiar dengan objek-objek Kubernetes untuk melakukan pembaruan tanpa terlebih dulu mengetahui *field-field* spesifik yang harus diperbarui:
+
+- `scale`: Melakukan *scaling* horizontal terhadap sebuah *controller* untuk menambah atau menghapus Pod dengan memperbarui jumlah replika dari *controller* tersebut.
+- `annotate`: Menambah atau menghapus anotasi sebuah objek.
+- `label`: Menambah atau menghapus label sebuah objek.
+
+Perintah `kubectl` juga mendukung perintah-perintah pembaruan berdasarkan salah satu aspek dari sebuah objek. Untuk tiap tipe objek yang berbeda, memperbarui sebuah aspek tertentu bisa berarti memperbarui sekumpulan *field* yang berbeda pula:
+
+- `set` `<field>`: Memperbarui sebuah satu aspek dari sebuah objek.
+
+{{< note >}}
+Pada Kubernetes versi 1.5, tidak semua perintah yang berdasarkan kata kerja berasosiasi dengan perintah yang berdasarkan aspek tertentu.
+{{< /note >}}
+
+*Tool* `kubectl` juga mendukung beberapa cara lain untuk memperbarui objek *live* secara langsung, meskipun cara-cara berikut membutuhkan pemahaman yang lebih tentang skema objek Kubernetes.
+
+- `edit`: Secara langsung mengedit konfigurasi mentah dari sebuah objek *live* dengan membuka konfigurasinya di sebuah editor.
+- `patch`: Secara langsung memodifikasi *field-field* spesifik dari sebuah objek *live* dengan menggunakan *patch string*. Untuk detil lebih lanjut mengenai `patch string`, lihat bagian tentang *patch* pada [Konvensi API](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#patch-operations).
+
+## Cara menghapus objek
+
+Kamu bisa menggunakan perintah `hapus` pada sebuah objek dari sebuah kluster:
```

Comment:
will we be always using `hapus` or `delete` since `delete` is already a keyword reserved for kubernetes command I think it will be easier to mention that, of course if you don't use another reason why introduce `hapus` as a command here

Reply:
oh, this should be `delete` since it's a command. My bad, will update this.